docs/docs_source/build_test_ship/shipping.rst

```diff
@@ -0,0 +1,146 @@
+
+..  _go live:
+..  _shipping:
+
+Shipping your Office Online integration
+=======================================
+
+..  spelling::
+
+    rollout
+    Rollout
+
+Once you believe you are ready to ship your integration, you should contact Microsoft and explain that you believe
```

Comment:
I'd like to avoid a conversational tone here; it often makes it more difficult for non-native English speakers to understand. I'm all for levity in general, just not in the docs. :smile: